packages/dds/tree/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -115,6 +118,11 @@ export type NodeChangeDecoder = (change: JsonCompatibleReadOnly) => NodeChangese
  * Changeset for a subtree rooted at a specific node.
  */
 export interface NodeChangeset {
+    /**
+     * The revision associated with `valueChange`, if any.
+     * This revision is not necessarily related to `fieldChanges`.
```

Comment:
By "not necessarily" do you mean "it is related be if the field changes don't themsleves specify a revision" or do you mean "it's independent (i.e., don't use it to interpret the field changes) but it may coincidentally be the same as that of the field changes"?
I'm assuming you mean the latter, in which case it may be clearer to make the added `revision` part of `ValueChange`.